Code/src/test/java/nl/utwente/group10/ui/components/blocks/FunctionBlockTest.java

```diff
@@ -20,9 +20,10 @@
      */
     @Before
     public void setUp() throws Exception {
+        //TODO: not entirely correct.
```

Comment:
(Int -> Int -> Int) is not the type of the (+) function, although it is not relevant in this testcase